personal/sarawozniak/reports/Zajecia3.Rmd

```diff
@@ -0,0 +1,83 @@
+---
+title: "Zajecia3"
+author: "Sara Wozniak"
+date: "25 04 2020"
+output: html_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = FALSE)
+```
+ 
+```{r}
+library(spData)
+library(dplyr)
+library(ggplot2)
+library(sf)
+```
+
+```{r}
+linestring_matrix = rbind(c(13, 53), c(14, 50))

```

Comment:
Dobrym zwyczajem jest trzymać się przypisywania zmiennych za pomocą "<-", a "=" wykorzystywać do argumentów wewnątrz funkcji. Sam mam to źle w prezentacji, więc muszę poprawić ;) .